### DISCUSSION TITLE
Antd website for Korean

### DISCUSSION THREAD

**Discussion by owjs3901** [ant-design/ant-design#52038] [Show and tell]
Hello.
I am aware of the existing anti-d policy.
I understand that the official website only supports English and Chinese.

However, I wanted to provide an anti-d website in Korean to Koreans.
So we are working on the project as follows.

[repo]
https://github.com/ant-design-korea/ant-design

[Released Website]
https://ant-design-korea.github.io/

I'd like to hear from the community if I can add Korean to the official website.

I know that there are still a lot of shortcomings and the translation has not progressed 100%.
However, it is not a small number of translations, but we are translating not only for Components but also for all menus such as Blog and Design.

Thank you.

![image](https://github.com/user-attachments/assets/51f6c7e5-c670-47d8-baf5-7846be55623c)
![image](https://github.com/user-attachments/assets/16b8feec-ed96-425b-b664-95b9dd26e78e)
![image](https://github.com/user-attachments/assets/2b2f2736-879e-4ab4-a31a-e3c6b6863612)

**Comment by petercat-assistant** [comment]
Thank you for your initiative in translating the Ant Design website into Korean. It's great to see the community contributing to make Ant Design more accessible to Korean users.

To add Korean to the official Ant Design website, you would typically need to collaborate with the Ant Design maintainers. Here are some steps you might consider:

1. **Community Feedback**: Engage with the Ant Design community to gather feedback and support for your translation project. This can help demonstrate the demand and usefulness of a Korean version.

2. **Collaboration with Maintainers**: Reach out to the maintainers of the Ant Design project. You can do this by opening a discussion or issue on the official GitHub repository to propose adding Korean as a supported language.

3. **Translation Quality**: Ensure that the translation is as complete and accurate as possible. Highlight the progress and quality of your translation work when communicating with the maintainers.

4. **Contribution Guidelines**: Follow any contribution guidelines provided by the Ant Design project for adding new languages or features.

5. **Pull Request**: If the maintainers agree, you may need to submit a pull request with your translation work to be reviewed and potentially merged into the official website.

Your existing repository and released website are excellent starting points to showcase your work. Keep up the good work, and best of luck with your project!

If you have any further questions or need assistance, feel free to ask.

<details>
<summary>🪧 Tips</summary>
For further assistance, please describe your question in the comments and @petercat-assistant to start a conversation with me.
</details>
